subiquity/common/errorreport.py

```diff
@@ -329,6 +333,20 @@ def ref(self):
             oops_id=self.oops_id,
             )
 
+    # with core24 these tag methods can be dropped for equivalent methods
+    # that will be on the report object
+    def get_tags(self) -> Set[str]:
+        """Return the set of tags."""
+        if "Tags" not in self.pr:
+            return set()
+        return set(self.pr["Tags"].split(" "))
+
+    def add_tags(self, tags: Iterable[str]) -> None:
+        """Add tags to the report. Duplicates are dropped."""
```

Comment:
Maybe a note that tags should not include spaces. That's not going to work very well.

Reply:
I'm inclined to leave it alone to stay as close as possible to python3-problem-report - thoughts?